.github/workflows/build.yml

```diff
@@ -25,24 +25,34 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
-          cache: 'yarn'
 
-      - name: yarn install
+      - name: Restore npm cache
+        uses: actions/cache@v2
+        with:
+          path: |
```

Comment:
I think it's discouraged to cache node_modules and cache .npm instead see here: https://github.com/actions/cache/blob/main/examples.md#node---npm

Reply:
Good catch! Thank you for the comment 👍